web-client/src/presenter/computeds/dashboardExternalHelper.test.js

```diff
@@ -68,7 +70,22 @@ describe('petitioner dashboard helper', () => {
     expect(result.showCaseSearch).toEqual(true);
   });
 
-  it('hides case search if defined user does not have privatePractitioner or irsPractitioner role', () => {
+  it('shows case search if defined user has peititioner role', () => {
```

Comment:
Nit: I think petitioner is spelled wrong?

Reply:
corrected the spelling...thx for catching it